cloud/blockstore/libs/storage/disk_agent/rdma_target.cpp

```diff
@@ -944,6 +944,7 @@ class TRdmaTarget final
 
     void Stop() override
     {
+        Server->DisconnectAll();
```

Comment:
Why can't we just call Stop here? Server->Stop() in turn will Stop all attached pollers

Reply:
You may argue that Server might potentially be used by someone else, not just TRdmaTarget, but closing all sessions/endpoints has pretty much the same effect in the end

Reply:
In fact there seems to be a race between peers to create a new session (since ConnectionPoller is still active) and DiskAgent actor to stop

Reply:
I stop all endpoints from listening so it's impossible to create new session.

Doing Server->Stop() seems like a better idea no need to add extra code.